homework/lesson2/sean.py

```diff
@@ -0,0 +1,9 @@
+name = input("What is your name? ")
+x = input("How many pizza you want? ")
+y = input("How many pepperonis do you want on YOUR FREAKING PIZZA? ")
+z = input("HOW MANY OLIVES DO YOU WANT ON YOU FREAKEDY FREAKING PIZZA!!! ")
+print(name + " orderd " + x + " FREAKING PIZZAS WITH " + y + " FREAKING PEPPERONIS AND " + z + " **** OLIVES.")
+print('''
+And you will get an extra FOOT LETTUCE ON YOUR PIZZA!
```

Comment:
nice job. your customer satisfaction is going to be through the roof lol.

Reply:
lol